docs/contributing.md

```diff
@@ -0,0 +1,143 @@
+# Contributors Guide
+
+This is a community driven project and everyone is welcome to contribute. The
+project is hosted at the [gmt-examples GitHub repository](https://github.com/GenericMappingTools/gmt-examples).
+
+The goal is to maintain a diverse community that's pleasant for everyone.
+**Please be considerate and respectful of others**. Everyone must abide by our
+[Code of Conduct](https://github.com/GenericMappingTools/gmt-examples/blob/main/CODE_OF_CONDUCT.md)
+and we encourage all to read it carefully.
+
+## Setting up your environment
+
+The following dependencies are required for building the GMT examples pages:
+
+- [gmt](https://docs.generic-mapping-tools.org/latest/#)
+- [myst-parser](https://myst-parser.readthedocs.io/en/latest/)
+- [sphinx](https://www.sphinx-doc.org/en/master/)
+- [sphinx_gmt](https://www.generic-mapping-tools.org/sphinx_gmt/latest/)
+- [sphinx_rtd_theme](https://sphinx-rtd-theme.readthedocs.io/en/stable/installing.html)
+
+Two options for installing the dependencies are [pip](#pip-setup) and [conda](#conda-setup).
+
+### Pip Setup
+
+These instructions rely on the [pip](https://pip.pypa.io/en/stable/) package
+installer, which can be used to install all dependencies except GMT.
+Follow the [GMT Install Guide](https://github.com/GenericMappingTools/gmt/blob/master/INSTALL.md)
+to install GMT. The Python dependencies can be installed using the
+`requirements.txt` from the base of the repository (you may wish to setup a
+[virtual environment](https://packaging.python.org/en/latest/guides/installing-using-pip-and-virtual-environments/#creating-a-virtual-environment) first):
+
+Unix/macOS:
+```
+python -m pip install -r requirements.txt
+```
+
+Windows:
+```
+py -m pip install -r requirements.txt
+```
+
+### Conda Setup
+
+These instructions rely on the [conda](https://docs.conda.io/en/latest/) package
+manager. Run the following from the base of the repository to create a new conda
+environment from the `environment.yml` file:
+
+```bash
+conda env create
+```
+
+Before building and testing the project, you have to activate the environment
+(you'll need to do this every time you start a new terminal):
+
+```bash
+conda activate gmt-examples
+```
+
+## Building the documentation
+
+Once you have [setup your environment](#setting-up-your-environment), you can
+build the documentation using:
+
+```bash
+    cd docs
+    make html
+```
+
+## GMT Examples Overview
+
+There are two main components to GMT examples project:
+
+* Gallery examples, with source material in the `docs/examples/gallery/` folder.
+* Tutorial examples, with source material in the `docs/examples/tutorials/` folder.
```

Comment:
Why not putting gallery and tutorial examples directly in the `docs` folder, i.e., `docs/gallery` and `docs/tutorials`.